resources/views/pages/user-shifts.html

```diff
@@ -1,8 +1,8 @@
 <form class="form-inline" action="">
     <input type="hidden" name="p" value="user_shifts">
+    <h1>%title% %add_link%</h1>
```

Comment:
I like the look of the title being out of the rows, but sadly it takes up way more space on the page if the filters are shown, which moves the shifts down. That's why this sadly feels impractical to me.